src/com/googlecode/jmxtrans/model/output/OpenTSDBWriter.java

```diff
@@ -113,6 +113,11 @@ protected void startOutput() throws IOException {
 	@Override
 	protected void sendOutput(String metricLine) throws IOException {
 		try {
+			
+			if (isDebugEnabled() || log.isDebugEnabled()) {
+				log.debug("Sending to server {}:{} line={}", host, port,metricLine);
```

Comment:
Sorry to be pedantic about it, but what if you were reading a book and the spacing was off? :wink: